index.html

```diff
@@ -1,57 +1,194 @@
+	<link rel="stylesheet" href="styles.css">
+	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+	<title>Portfolio Project</title>
 </head>
 <body>
-    <header>
-    <nav class="nav-bar">
-        <a href="#" onClick="return false;">
-        <p class="title">Welcome</p>
-        </a>
-        <a class="icon">
-            <i class="fa fa-bars"></i>
-        </a>
-    </nav>
-    </header>
-    <section class="main-container">
-    <div class="headline">
-        <div class="intro-txt">
-            <h1 class="wave-txt">Hey there. I'm Lucas</h1>
-            <h2 class="desc-txt">I'm a software developer</h2>
-        </div>
-        <div class="main-headline-txt">
-            <p>
-                I can help you build a product, feature or website Look through some of my work and experience! 
-                If you like what you see and have a project you need coded, don't hestiate to contact me.
-            </p>
-        </div>
-        <div class="media-imgs">
-            <ul>
-            <li>
-                <img src="imgs/twitter.svg" alt="Twitter img">
-            </li>
-            <li>
-                <img src="imgs/Linkedin-icon.svg" alt="Linkedin img">
-            </li>
-            <li>
-                <img src="imgs/medium.svg" alt="Medium img">
-            </li>
-            <li>
-                <img src="imgs/github.svg" alt="Github img">
-            </li>
-            <li>
-                <img src="imgs/angellist.svg" alt="Angellist img">
-            </li>
-            </ul>
-        </div>
-    </div>
-    </section>
-    
+	<header>
+	<nav class="nav-bar">
+		<a href="#" onClick="return false;">
+		<p class="title">Welcome</p>
+		</a>
+		<a class="icon">
+			<i class="fa fa-bars"></i>
+		</a>
+	</nav>
+	</header>
+	<section class="main-container">
+		<div class="headline">
+			<div class="intro-txt">
+				<h1 class="wave-txt">Hey there. I'm Lucas</h1>
+				<h2 class="desc-txt">I'm a software developer</h2>
+			</div>
+			<div class="main-headline-txt">
+				<p>
+					I can help you build a product, feature or website Look through some of my work and experience! 
+					If you like what you see and have a project you need coded, don't hestiate to contact me.
+				</p>
+			</div>
+			<div class="media-imgs">
+				<ul>
+				<li>
+					<img src="imgs/twitter.svg" alt="Twitter img">
+				</li>
+				<li>
+					<img src="imgs/Linkedin-icon.svg" alt="Linkedin img">
+				</li>
+				<li>
+					<img src="imgs/medium.svg" alt="Medium img">
+				</li>
+				<li>
+					<img src="imgs/github.svg" alt="Github img">
+				</li>
+				<li>
+					<img src="imgs/angellist.svg" alt="Angellist img">
+				</li>
+				</ul>
+			</div>
+		</div>
+	</section>
+	<section class="works-sec">
+		<div class="works-title-flex">
+			<h2 class="wave-txt" id="works-txt-id">My Recent Works</h2>
+			<img class="div-img" src="imgs/indicator.png" alt="div-img">

```

Comment:
- [ ] The picture won't open on some computers because the name of the picture file is not correct. Some operating systems like Linux are case-sensitive. Please change the name of the file to **I**ndicator.png so that it loads properly.

Reply:
good catch, didn't see that one !